src/main/java/at/hannibal2/skyhanni/features/bingo/MinionCraftHelper.kt

```diff
@@ -283,14 +281,21 @@ class MinionCraftHelper {
             }
             element
         }
+        event.transform(26, "#player.bingoSessions") { element ->
```

Comment:
We could delete the transform call above that got added at config version 19. Not sure if we want to set a president in deleting migrations this way, but the result is the very same, right?

Reply:
I agree in not deleting migrations, but as these migrations are the same and don't actually move anything this is completely fine